wave_vision/src/matcher/brute_force_matcher.cpp

```diff
@@ -5,12 +5,17 @@ namespace wave {
 
 // Default constructor. Struct may be default or user defined.
 BruteForceMatcher::BruteForceMatcher(const BFMatcherParams &config) {
+    bool cross_check;
+
     // Ensure parameters are valid
     this->checkConfiguration(config);
 
+    // Cross_check must be the opposite of use_knn
+    cross_check = !config.use_knn;
+
     // Create cv::BFMatcher object with the desired parameters
     this->brute_force_matcher =
-      cv::BFMatcher::create(config.norm_type, config.cross_check);
+      cv::BFMatcher::create(config.norm_type, cross_check);
 
     // Store configuration parameters within member struct
     this->current_config = config;
```

Comment:
@leokoppel  the reason this wasn't in an initializer list is because I want to check the config for validity before continuing on. If the config is invalid then there is an `std::invalid_argument` thrown from the `checkConfiguration` method.